gravitee-apim-portal-webui-next/src/app/api-details/api-details.component.html

```diff
@@ -15,7 +15,27 @@
     limitations under the License.
 
 -->
-<mat-card appearance="outlined">
+<ng-template #separatorTemplate>
```

Comment:
What about moving all related to the breadcrumb to a separate component? It could be reused in other scenarios

Reply:
Done